app/script/localization/translations/webapp-de.js

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-'use strict';
+'use=strict';
```

Comment:
strange..

Reply:
I think crowdin adds the 'use=strict'. In the ported node version of the translation script we dont have a replace for those, as this statement isn't needed anymore.